rllib/agents/slateq/tests/test_slateq.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+
+import ray
+import ray.rllib.agents.slateq as slateq
+from ray.rllib.examples.env.recsim_recommender_system_envs import \
+    LongTermSatisfactionRecSimEnv
+from ray.rllib.utils.test_utils import check_compute_single_action, \
+    check_train_results, framework_iterator
+
+
+class TestSlateQ(unittest.TestCase):
```

Comment:
Hey, sorry, but this would take another day or so to write due to the 2 different models and complex loss math used. 

Could we defer such a test for later? Note that this algo was originally contributed to Ray by Henry (@desktable ) in late 2020 and this PR only adds a few enhancements to it to make the Q-loss and -value terms more stable and to enable exploration. It will require some more investigation by us in the coming weeks to fully claim support for this and confirm "hard-task" learning capabilities.

Reply:
But I think it's great that you want to go through the paper to double check with our implementation! Please let me know, if you find more issues with our math.